tensorflow_datasets/text/cnn_dailymail.py

```diff
@@ -219,14 +225,14 @@ class CnnDailymail(tfds.core.GeneratorBasedBuilder):
           name='bytes',
           description=('Uses byte-level text encoding with '
                        '`tfds.features.text.ByteTextEncoder`'),
-          text_encoder_config=tfds.features.text.TextEncoderConfig(
+        text_encoder_config=tfds.features.text.TextEncoderConfig(
```

Comment:
Could you run `pylint` to format your workspace ? For instance why removing spaces here ?

Reply:
I run and fixed